content/blog/2021/10/2021-10-31-hacktoberfest-results-2021.adoc

```diff
@@ -0,0 +1,88 @@
+
+link:https://github.com/highflyer910[Thea Mushambadze] created the new Jenkins logo for Georgia logo.
+
+image:/images/logos/georgia/georgia.png[ height=200 ]
+
+Thanks very much!
+
+== Plugin Docs Migration to GitHub
+
+The Jenkins project started the migration of plugin documentation to GitHub almost two years ago.
+Hacktoberfest 2021 identified 40 candidate plugins to migrate their documentation to GitHub.
+link:https://github.com/Dan-Heath[Dan Heath],
+link:https://github.com/Mr-DG-Wick[Deepak Gupta],
+link:https://github.com/avinashupadhya99[Avinash Upadhyaya K R],
+link:https://github.com/rajanssingh[Rajan Kumar Singh], and
+link:https://github.com/dheerajodha[Dheeraj Singh Jodha] provided a total of 29 pull requests migrating plugin documentation to GitHub.
+
+Special thanks to the contributors and to the plugin maintainers that merged and released documentation migration pull requests.
+Eight plugins have fully migrated their documentation to GitHub through Hacktoberfest contributions.
+An additional thirteen plugins have merged the documentation change and will complete the migration with their next plugin release.
+
+The plugin documentation migration was coordinated through a link:https://docs.google.com/spreadsheets/d/1xxz6v-N6h17PI9Qu9uKrWaiKdI94StfeTfzWbwIYsX8/edit?usp=sharing[worksheet].
+A link:https://www.youtube.com/watch?v=NU2g373wHNo&t=2331s[tutorial video] describing the documentation migration process is also available.
+
+== Implementing Content Security Policy
+
+Wadeck Follonier provided a link:https://www.youtube.com/watch?v=NU2g373wHNo&t=3369s[tutorial] to prepare Jenkins for a broader implementation of Content Security Policy.
+Six contributors provided pull requests to Jenkins core to create separate JavaScript files from JavaScript that was previously implemented inside the Jenkins 'jelly' files.
+Moving JavaScript into separate files prepares Jenkins to implement Content Security Policy protections against JavaScript embedded in HTML.
+
+== Jenkins Architecture Diagrams
+
+Angélique Jard provided architectural diagrams showing a link:/doc/developer/architecture/[dataflow view of Jenkins] and a link:/doc/developer/architecture/model/[high level view of Jenkins].
+Those diagrams help others understand the internal structure of Jenkins and how it interacts with users and with other systems.
+
+image:/images/developer/architecture/jenkins-dataflow.png[]
+
+== Modernizing Plugins Video Series
+
+Darin Pope created link:https://www.youtube.com/watch?v=Fev8KfFsPZE[five videos] illustrating some of the small steps that a new contributor can use to help modernize a plugin.
+Additional plugin modernization and adoption ideas are included in the link:https://docs.google.com/document/d/1PKYIpPlRVGsBqrz0Ob1Cv3cefOZ5j2xtGZdWs27kLuw/edit?usp=sharing["Contributing to Open Source" workshop] from DevOps World 2021.
+
+.Modernizing Jenkins Plugins
+video::Fev8KfFsPZE[youtube, width=640, height=363]
```

Comment:
oh cool, so there is a video shortcut? should it have double colons?

Reply:
Yes, there's a video shortcut, and it needs double colons to work as far as the other examples in the source files.  See https://www.jenkins.io/blog/2021/08/19/git-credentials-binding-work-report/ and https://www.jenkins.io/blog/2021/08/02/cloudevents-plugin-phase-I/ for examples